src/commit.c

```diff
@@ -279,15 +280,11 @@ int apk_solver_commit_changeset(struct apk_database *db,
 	foreach_array_item(change, changeset->changes) {
 		count_change(change, &prog.total);
 		if (change->new_pkg)
-			size_diff += change->new_pkg->installed_size / 1024;
```

Comment:
`size_diff` can now overflow on 32-bit systems at 2 GB. Please keep this division, or use `int64_t` or similar data type.

Reply:
`off_t`?

Reply:
Used `off_t` and `llabs` instead, not sure how portable is though.

Reply:
We already use `long long` and `off_t` in some places. So I am ok with these. `llabs` should be ok too.